build.sbt

```diff
@@ -8,10 +8,10 @@ ThisBuild / scalacOptions := Seq(
   "-encoding", "UTF-8", "-target:jvm-1.8", "-deprecation",
   "-feature", "-unchecked", "-language:implicitConversions", "-language:postfixOps")
 
-val languageToolVersion = "5.9"
-val awsSdkVersion = "1.11.999"
+val languageToolVersion = "6.0"
+val awsSdkVersion = "1.12.416"
 val capiModelsVersion = "17.4.0"
```

Comment:
You should keep the version of capi models in sync with the version that the capi client pulls in. It would be nice if this happened automatically